examples/options/src/detector.cpp

```diff
@@ -35,6 +35,9 @@ detector::detector() : interface("Detector Options") {
         "digitization-file",
         po::value(&digitization_file)->default_value(digitization_file),
         "Digitization file");
+    m_desc.add_options()("use-detray-simulation",
+                         po::bool_switch(&use_detray_simulation),
```

Comment:
Don't forget the `->default_value(...)` part. :wink:

Reply:
OK. . It is supposed to be `true` considering that the acts geometry is going to be the standard, but I set it to `false` as `use-detray-detector` is set to `false` as well. We can change this once we fully deprecate the csv geometry in the near future